hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -141,7 +140,7 @@
   // Setup for generating OTP images.
   gen_otp_images_cfg_dir: "{proj_root}/hw/ip/otp_ctrl/data"
   gen_otp_images_cmd: "{proj_root}/util/design/gen-otp-img.py"
-  gen_otp_images_cmd_opts: ["--quiet", "--seed {seed}"]
```

Comment:
Cannot use value `seed` anymore because it is over-written by SV seed value.

Reply:
Thanks for helping debug this issue @msfschaffner ! I still added the `--img-seed` here to randomize it. Please let me know if you have any concerns. Or it is not necessary.

Reply:
This should be ok (it is better to randomize it instead of not randomizing it at all).

Reply:
Enforcing a build time selected seed for a runtime command is not a good idea. Why does this need to match thd build seed?

Reply:
Currently the otp_image_cmd requires to use the same seed from `gen-otp-mmap` command.
@msfschaffner Do you think we can avoid this issue by letting the `gen-otp-img` script finds the correct seed itself from generated files (such as otp_ctrl_part_pkg.sv)? I think that file should have the correct seed?